config/locales/en_US.yml

```diff
@@ -107,7 +107,7 @@ en_US:
         mico_task: "Micro Task Assignment?"
         review_reviewers: "Reviews visible to all other reviewers?"
         calibrate_training: "Calibration for training?"
-        reputation_algorithm: "Reputation Alogorithm?"
+        reputation_algorithm: "Reputation Algorithm?"
```

Comment:
--> "Reputation algorithm".  Notice that the other strings capitalize only the first word.